src/main/scala/chisel3/ltl/LTL.scala

```diff
@@ -257,7 +257,7 @@ object Sequence {
     *
     * See `SequenceAtom` for more details.
     */
-  def apply(atoms: SequenceAtom*): Sequence = {
+  def apply(atoms: SequenceAtom*)(implicit sourceInfo: SourceInfo): Sequence = {
```

Comment:
Should we use macro for this? wdyt @jackkoenig

Reply:
Class `Sequence` does not define an `apply` method, so fortunately there is no need!